pom.xml

```diff
@@ -129,8 +129,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.8.1</version>
 				<configuration>
-					<source>1.7</source>
-					<target>1.7</target>
+					<source>1.8</source>
+					<target>1.8</target>
```

Comment:
This is a breaking change, should we also bump the major version to 4.0.0?

Reply:
IMO, no. Many projects if not most of them (i.e. commons-pool) did not bump the major version when they moved to java 8.

Reply:
I mean it's a breaking change by definition, as in JDK 7 will not be supported anymore, which is totally fine by me btw but I don't know if there are consumers depending on that. Bumping to 4.x isn't necessary, it's just the usual versioning scheme but there is not strict rule that we have to follow it.

I am okay with the change anyway, but if we do this we should drop this bit from the CI: https://github.com/redis/jedis/blob/master/.circleci/config.yml#L20-L22

Reply:
Agreed.

PS: that piece is meaningless because of https://github.com/redis/jedis/blob/master/.circleci/config.yml#L73-L74

Reply:
That piece is meaningless because of https://github.com/redis/jedis/blob/master/.circleci/config.yml#L73-L74 

However, I agree to remove. Will do it separately.